jd4/daemon.py

```diff
@@ -103,12 +109,16 @@ def __init__(self, session, request, ws):
             judge_tasks.append(loop.create_task(pool_judge(package, case)))
         for index, judge_task in enumerate(judge_tasks):
             status, score, time_usage_ns, memory_usage_bytes, stderr = await judge_task
+            case = {'status': status,
+                    'score': score,
+                    'time_ms': time_usage_ns // 1000000,
+                    'memory_kb': memory_usage_bytes // 1024}
+            if self.type == 1:
+                case['judge_text'] = stderr.decode(encoding='utf-8', errors='replace')
+            else:
+                case['judge_text'] = ''
```

Comment:
I thought judge_text was optional.
Since it's mandatory, let's revert this ^_^T_T